scholarx/archive/index.html

```diff
@@ -101,7 +101,9 @@ <h6>UNIVERSITY</h6>
             <div class="search-filter">
                 <input id="search" type="text" class="form-control rounded search-bar" placeholder="Search" aria-label="Search" aria-describedby="search-addon" />
             </div>
-            <!--mentor details-->
+            <!--no results message-->
+            <div id="noResultsMessage" class="no-results-page" style="display: none;"><img src="no-results-found.png" alt="No Results Found image"></div>
```

Comment:
Let's just use a text and font awesome icon. https://fontawesome.com/search?q=search&o=r